tests/kubeless/kubeless-tests.go

```diff
@@ -230,35 +237,51 @@ func randomString(n int) string {
 	return string(b)
 }
 
-func cleanup(namespace, functionName string) {
-	log.Println("Cleaning up")
-	deleteFun(namespace, functionName)
-	deleteNamespace(namespace)
+func cleanup(namespace, functionName string) error{
+	log.Info("Cleaning up")
+	err:=deleteFun(namespace, functionName)
```

Comment:
`go fmt` is not too happy about that syntax.

Reply:
fixed